README.md

```diff
@@ -25,20 +24,21 @@ $ kubectl create -f example/deployment.yaml
 deployment "etcd-operator" created
 ```
 
-etcd operator will create a Kubernetes *Third-Party Resource* (TPR) called "etcd-cluster".
+etcd operator will create a Kubernetes *Third-Party Resource* (TPR) called "etcd-cluster" automatically.
 
 ```bash
 $ kubectl get thirdpartyresources
 NAME                      DESCRIPTION             VERSION(S)
 etcd-cluster.coreos.com   Managed etcd clusters   v1
-```
 
 ## Create an etcd cluster
 
 ```bash
 $ kubectl create -f example/example-etcd-cluster.yaml
 ```
 
+This will create 3 members etcd cluster eventually:
```

Comment:
A 3 member etcd cluster will be created. (remove eventually? it is too verbose i feel)